src/main/kotlin/Utilities/FileIo.kt

```diff
@@ -1,7 +1,22 @@
 package Utilities
 
-class FileIo {
-    fun readResource(fileName: String): String {
-        return javaClass.classLoader.getResource(fileName).readText(Charsets.UTF_8)
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.Paths
+
+class FileIo : FileInterface {
+    override fun readResource(filePath: String): String {
+        val path = Paths.get(filePath)
+        val file = File(path.toUri())
+        return file.readText(Charsets.UTF_8)
+    }
+
+    override fun writeResource(filePath: String, resource: String) {
```

Comment:
Sorry, I think I led you down the wrong path few weeks ago when we paired. [Myles mentioned](https://8thlight.slack.com/archives/C02G3B5TS/p1640097454271000?thread_ts=1640096535.270900&cid=C02G3B5TS) that files should be static in the resources directory. As he mentioned, I think it's better that you create a directory for writing files at runtime. 

I would also revert `readResource` to read static files from the resources directory. This may mean that you need a new read function for reading files you wrote outside of the resources directory.

Reply:
The two functions, `readResource` and `writeResource` are for reading/writing files at runtime. I've added a new function `readFile` to read static files from the resources directory. I've also included another `resources` file in the following directory path `src/main/kotlin/resources`, and the same in the test directory.